src/test/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterControllerTest.java

```diff
@@ -41,11 +46,34 @@
     @MockBean
     private AuthTokenValidator tokenValidator;
 
-    private static final String LETTER_JSON = "{"
-        + "\"template\": \"abc\","
-        + "\"values\": { \"a\": \"b\" },"
-        + "\"type\": \"typeA\""
-        + "}";
+    private static String LETTER_JSON;
+    private static String LETTER_WITHOUT_DOCUMENTS_JSON;
+    private static String LETTER_WITHOUT_TYPE_JSON;
+    private static String LETTER_WITHOUT_DOC_TEMPLATE_JSON;
+    private static String LETTER_WITHOUT_DOC_TEMPLATE_VALUES_JSON;
+    private static String LETTER_WITH_MORE_THAN_TEN_DOCS_JSON;
+
+    @BeforeClass
+    public static void setUp() throws IOException {
+        LETTER_JSON = new String(
+            readAllBytes(Paths.get("src/test/resources/letter.json"))
+        );
+        LETTER_WITHOUT_DOCUMENTS_JSON = new String(
+            readAllBytes(Paths.get("src/test/resources/letter-without-doc.json"))
+        );
+        LETTER_WITHOUT_TYPE_JSON = new String(
+            readAllBytes(Paths.get("src/test/resources/letter-without-type.json"))
+        );
+        LETTER_WITHOUT_DOC_TEMPLATE_JSON = new String(
+            readAllBytes(Paths.get("src/test/resources/letter-without-template.json"))
+        );
+        LETTER_WITHOUT_DOC_TEMPLATE_VALUES_JSON = new String(
+            readAllBytes(Paths.get("src/test/resources/letter-without-template-values.json"))
+        );
+        LETTER_WITH_MORE_THAN_TEN_DOCS_JSON = new String(
+            readAllBytes(Paths.get("src/test/resources/letter-with-multiple-docs.json"))
+        );
+    }
```

Comment:
definitely need better resource reading ;). perhaps we should name constants as file names then call some sort of setup function for individual test to read it's contents.